metrics/lib/send_results.sh

```diff
@@ -36,7 +36,7 @@ COR_DEFAULT_PATH="/usr/bin"
 COR_DEFAULT_NAME="cc-oci-runtime"
 RUNC_DEFAULT_PATH="/usr/bin"
 RUNC_DEFAULT_NAME="docker-runc"
-CC_RUNTIME_DEFAULT_PATH="/usr/local/bin"
+CC_RUNTIME_DEFAULT_PATH="/usr/bin"
```

Comment:
@chavafg thanks for the observation, @grahamwhaley I can add that in this commit :)